Hush/Assets/Scripts/Weapon/IWeapon.cs

```diff
@@ -1,9 +1,11 @@
 using System.Collections.Generic;
 
-public interface IWeapon {
+public interface IWeapon
+{
+    string WeaponType { get; }
```

Comment:
Wouldn't we prefer an enum? If not we might want a static getter class to get the possible string values

```
public static class WeaponTypes {
    public static string Sword => "Sword";
}
```

Reply:
I guess accessing it through the specific class is also good, just usually prefer to have everything in 1 place so you can see your options!

Reply:
yeah, I was considering that, but I didn't want to change too much since this is just a hotfix

Reply:
All good can do later!